src/models/hosts.ts

```diff
@@ -1,3 +1,9 @@
 export interface Host {
-  id: string;
+  name: string;
+  ip: string;
```

Comment:
The types of the fields of `Host` should match whatever the server returns.

Reply:
done